concepts/05 Widgets/LoadPanel/05 Show and Hide Using the API.md

```diff
@@ -92,47 +92,97 @@ With Angular, AngularJS, or Knockout, use a different technique. Bind the [visib
         // ...
     })
 
-##### AngularJS
+##### Vue
+
+    <template>
```

Comment:
Please add tab names to all Vue and React examples.

Reply:
We decided to do it later